azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/internal/BaseAzureComputeApiLiveTest.java

```diff
@@ -74,6 +83,17 @@ protected String getEndpoint() {
       return endpoint;
    }
 
+   protected String getSubscriptionId() {
+      String subscriptionid = null;
+      String endpoint = null;
+      endpoint = getEndpoint();
+      if (endpoint != null) {
+         subscriptionid = endpoint.substring(endpoint.lastIndexOf("/") + 1);
+      }
+      assertNotNull(subscriptionid);
+      return subscriptionid;
+   }
```

Comment:
This is actually not needed, since we have updated `VirtualMachineApi.create()`'s payload as `@Payload("%7B\"location\":\"{location}\",\"tags\":%7B%7D,\"properties\":{properties}%7D")`. Hence we are still using `BindToJsonPayload`.